src/modules/translate/translateApi.js

```diff
@@ -23,6 +20,7 @@ export const fetchNnTranslation = ({ id, ...articleContents }) =>
     body: JSON.stringify({
       token: config.npkToken,
       guid: config.ndlaEnvironment + '_' + id,
+      prefs: { x: true },
```

Comment:
Kanskje slenge på en kommentar for å nevne hva denne gjør siden navnet var litt vanskelig å forstå? :smile:

Reply:
Kan eg godt.